enterprise/server/raft/filestore/filestore.go

```diff
@@ -276,6 +310,49 @@ func (pmk *PebbleKey) parseVersion3(parts [][]byte) error {
 	return nil
 }
 
+func (pmk *PebbleKey) parseVersion4(parts [][]byte) error {
+	digestFunctionString := "1"
```

Comment:
nit: default to empty string. it may look to readers like this default value has some significance when in fact it's always overridden below.